### ISSUE TITLE
blockToHTML without encoding

### ISSUE THREAD

**Issue by jonzlin95** [HubSpot/draft-convert#80]
Hi,

Not sure if this is in-scope with draft-convert, but I'm trying to get a code block into my editor that renders as pure HTML. The editor is a pure internal tool (so we're not worried about malicious actors). 

Things I've tried

```javascript
{blockToHTML: (block) => {
    if (block.type === 'code-block') {
      return <div dangerouslySetInnerHTML={{ __html: block.text }}></div>
    }
}
```

Can't set children and innerHTML

```javascript
{blockToHTML: (block) => {
    if (block.type === 'code-block') {
      return {
        start: block.text,
        end: ''
      }
    }
}
```

Outputs: Correct output + text is repeated

It would be nice if there's a non-default way to avoid encoding the blocks so we can pass unsafe HTML in convertToHTML.

**Comment by jonzlin95** [comment]
I noticed that blockToHTML has support for string return values from

convertToHTML.js (96-106)
```javascript
    const blockHTML = getBlockTags(
      getBlockHTML(block)
    );

    let html;

    if (typeof blockHTML === 'string') {
      html = blockHTML;
    } else {
      html = blockHTML.start + innerHTML + blockHTML.end;
    }
```

But there is an invariant in getBlockTags.js (28-32)
``` javascript
  invariant(
    (Object.prototype.hasOwnProperty.call(blockHTML, 'start') && Object.prototype.hasOwnProperty.call(blockHTML, 'end'))
     || typeof(blockHTML,
    'convertToHTML: received block information without either a ReactElement or an object with start/end tags'
  );
```

which prevents a string from ever being used. I solved my problem by simply changing the invariant to also allow strings, but I haven't submitted a PR since it'd be a major API change of sorts. Just curious as to your thoughts @benbriggs if this is by design, or if you're planning to drop all support for strings in the future?

Thanks!

**Comment by benbriggs** [comment]
@jonzlin95 good catch! There should be an early return in `getBlockTags` when strings are passed to it to allow strings. I'll push up a PR now. Thank you!

**Comment by benbriggs** [comment]
published in 1.4.8

**Comment by jonzlin95** [comment]
Thanks for the quick response Ben. As usual pleasure using your library, and great work! 👍
